tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5268,6 +5268,29 @@ def _impl(ctx):
         provides = ["profile"],
     )
 
+    index_while_building_feature = feature(
+            name = "index_while_building",
+            enabled = True,
+            flag_sets = [
+                flag_set(
+                    actions = [
+                        ACTION_NAMES.c_compile,
+                        ACTION_NAMES.cpp_compile,
+                        ACTION_NAMES.objc_compile,
+                        ACTION_NAMES.objcpp_compile,
+                    ],
+                    flag_groups = [
+                        flag_group(
+                            flags = ["-index-store-path", "%{index_store_path}", "-index-ignore-system-symbols"],
```

Comment:
I added the `ignore` flag here as otherwise we will get a large amount of duplicated index information per each objc file compilation. For one file I looked at this resulted in 300kb of index data vs 7MB. We should probably do the same with the Swift indexing and somehow come up with a pass for the system headers.

Reply:
Thinking about it, I should pull this out and just leave it as an opt in flag that can be set globally.